howto/unicode.po

```diff
@@ -274,31 +274,33 @@ msgid "It can handle any Unicode code point."
 msgstr "Il peut gérer n'importe quel point de code Unicode."
 
 #: ../Doc/howto/unicode.rst:138
+#, fuzzy
 msgid ""
-"A Unicode string is turned into a sequence of bytes containing no embedded "
-"zero bytes.  This avoids byte-ordering issues, and means UTF-8 strings can "
-"be processed by C functions such as ``strcpy()`` and sent through protocols "
-"that can't handle zero bytes."
+"A Unicode string is turned into a sequence of bytes that contains embedded "
```

Comment:
Celle ci est bien `fuzzy` et trop grosse pour etre modifiée in github. Ca se fera a part :)